.github/workflows/test_build.yml

```diff
@@ -0,0 +1,19 @@
+name: Test build
+
+on: pull_request
+
+jobs:
+  test-deploy:
+    name: Test build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          cache: npm
+
+      - name: Install dependencies
+        run: npm ci
+      - name: Test build website
+        run: yarn run build
```

Comment:
This is the only occurrence of `yarn`, shouldn't this have being `npm`?

Reply:
Oh good catch! I even changed this line to add the `run` but missed the `yarn` 🤦